content/200-concepts/100-components/02-prisma-client/060-full-text-search.mdx

```diff
@@ -170,10 +170,13 @@ For the full range of supported operations, see the [MySQL full text search docu
 
 ### PostgreSQL
 
-To speed up your full-text queries, you should add an index to your database. Prisma Migrate currently doesn't support adding search indices in PostgreSQL, so this should be added in SQL. For example, the following SQL statement would add an index called `post_body_index` on the `body` column of the `posts` table:
+To speed up your full-text queries, you should add an index to your database. Prisma Schema currently doesn't support defining and Prisma Migrate doesn't support creating search indices, but you can easily add one yourself. The following example adds a `GIN` index to the table `posts` on the field `body`:
```

Comment:
The index would be created in the database, but the query executed/ generated by Prisma Client doesn't use the index. I'd prefer we wait till that issue is resolved in Prisma Client.

Related issue: https://github.com/prisma/prisma/issues/8950

Reply:
The current state is exactly the same, why should we wait on making the wording more exact about what we already document?